### PR TITLE
add postList

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
[DEMO LINK](https://DidenkoDenis306.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/components/CommentInfo/CommentInfo.tsx [hunk 1]
```diff
@@ -1,5 +1,30 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
```
Comment (ruslanpashkov) [inline src/components/CommentInfo/CommentInfo.tsx:6]: we can also use `interface` here

**Review by ruslanpashkov — APPROVED** [review]

File: src/components/CommentInfo/CommentInfo.tsx (hunk 1 above)
Comment (33Orange) [reply src/components/CommentInfo/CommentInfo.tsx:6]: > we can also use `interface` here

matter of taste

File: src/components/PostInfo/PostInfo.tsx [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+      </p>
+
+      <hr />
+
+      {comments.length
+        ? (
+          <CommentList comments={comments} />
+        ) : (
+          <b data-cy="NoCommentsMessage">No comments yet</b>
+        )}
```
Comment (33Orange) [inline src/components/PostInfo/PostInfo.tsx:48]:
```suggestion
      {comments.length
        ? (<CommentList comments={comments} />) 
        : (<b data-cy="NoCommentsMessage">No comments yet</b>)
      }
```

**Review by 33Orange — APPROVED** [review]
Well done !

**Pull request closed without merging**